### PR TITLE
Better upload errors client-side

### PR DESCRIPTION
**Pull request by mbarton**
Improves the error handling code for direct uploads to log the status code and response text where available.

Also breaks down 'error uploading video' into errors at each specific part request.

### CODE REVIEW

File: public/video-ui/src/actions/UploadActions/getUploads.js [hunk 1]
```diff
@@ -10,8 +10,6 @@ function runningUploads(uploads) {
 }
 
 function errorGettingUploads(error) {
-  Logger.error(error);
```
Comment (Reettaphant) [inline public/video-ui/src/actions/UploadActions/getUploads.js]: Why is this line removed?

Comment (mbarton) [inline public/video-ui/src/actions/UploadActions/getUploads.js]: We now print useful info about the error into the big red UI box rather than on the console

Comment (Reettaphant) [inline public/video-ui/src/actions/UploadActions/getUploads.js]: Sounds sensible. Would be useful to do this for other error reporting as well later?

Comment (mbarton) [inline public/video-ui/src/actions/UploadActions/getUploads.js]: Yeah I've held off doing that just to keep the PR small
